package.json

```diff
@@ -15,7 +15,7 @@
         "artedi": "1.1.1",
         "assert-plus": "0.1.5",
         "backoff": "2.3.0",
-        "bignum": "0.6.0",
+        "bignum": "0.13.0",
```

Comment:
"bunyan": "0.22.1",

Really should update bunyan to latest 1.x: ^1.8.12

Reply:
will do